client/src/main/java/ObjType.java

```diff
@@ -328,173 +328,195 @@ private int[] method8795(@OriginalArg(1) int arg0, @OriginalArg(2) int[] arg1) {
+				this.anIntArray798[opcode - 100] = packet.g2();
+			} else if (opcode == 110) {
+				this.anInt10125 = packet.g2();
+			} else if (opcode == 111) {
+				this.anInt10131 = packet.g2();
+			} else if (opcode == 112) {
+				this.lb = packet.g2();
+			} else if (opcode == 113) {
+				this.anInt10100 = packet.g1b();
+			} else if (opcode == 114) {
+				this.anInt10109 = packet.g1b() * 5;
+			} else if (opcode == 115) {
+				this.anInt10121 = packet.g1();
+			} else if (opcode == 121) {
+				this.anInt10143 = packet.g2();
+			} else if (opcode == 122) {
+				this.anInt10083 = packet.g2();
+			} else if (opcode == 125) {
+				this.anInt10147 = packet.g1b() << 2;
+				this.anInt10146 = packet.g1b() << 2;
+				this.anInt10118 = packet.g1b() << 2;
+			} else if (opcode == 126) {
+				this.anInt10113 = packet.g1b() << 2;
+				this.anInt10116 = packet.g1b() << 2;
+				this.anInt10136 = packet.g1b() << 2;
+			} else if (opcode == 127) {
+				this.anInt10129 = packet.g1();
+				this.anInt10112 = packet.g2();
+			} else if (opcode == 128) {
+				this.anInt10081 = packet.g1();
+				this.anInt10082 = packet.g2();
+			} else if (opcode == 129) {
+				this.anInt10122 = packet.g1();
+				this.anInt10089 = packet.g2();
+			} else if (opcode == 130) {
+				this.anInt10093 = packet.g1();
+				this.anInt10091 = packet.g2();
+			} else if (opcode == 132) {
+				local202 = packet.g1();
 				this.anIntArray800 = new int[local202];
 				for (local212 = 0; local212 < local202; local212++) {
-					this.anIntArray800[local212] = arg0.g2();
+					this.anIntArray800[local212] = packet.g2();
 				}
-			} else if (arg1 == 134) {
-				this.anInt10138 = arg0.g1();
-			} else if (arg1 == 139) {
-				this.anInt10114 = arg0.g2();
-			} else if (arg1 == 140) {
-				this.anInt10144 = arg0.g2();
-			} else if (arg1 == 249) {
-				local202 = arg0.g1();
-				if (this.aHashTable44 == null) {
-					local212 = IntUtils.clp2(local202);
-					this.aHashTable44 = new HashTable(local212);
+			} else if (opcode == 134) {
+				this.anInt10138 = packet.g1();
+			} else if (opcode == 139) {
+				this.anInt10114 = packet.g2();
+			} else if (opcode == 140) {
+				this.anInt10144 = packet.g2();
+			} else if (opcode == 249) {
+				int size = packet.g1();
+				if (this.params == null) {
+					int capacity = IntUtils.clp2(size);
+					this.params = new HashTable(capacity);
 				}
-				for (local212 = 0; local212 < local202; local212++) {
-					@Pc(554) boolean local554 = arg0.g1() == 1;
-					@Pc(558) int local558 = arg0.g3();
-					@Pc(567) Linkable local567;
-					if (local554) {
-						local567 = new StringNode(arg0.gjstr());
+
+				for (int count = 0; count < size; count++) {
+					@Pc(554) boolean isString = packet.g1() == 1;
+					@Pc(558) int key = packet.g3();
+
+					@Pc(567) Linkable node;
+					if (isString) {
+						node = new StringNode(packet.gjstr());
 					} else {
-						local567 = new IntNode(arg0.g4());
+						// Forcing 457 (shard return) to be 105 for the Rune Minotaur Pouch
```

Comment:
Breaks the crafting interface for rune minotaur. Shows the lit non-grey version of the pouch even when you can craft it. Need to look into this

Reply:
From what I can tell that interface doesn't use 457 so I'm not sure why that's happening

Reply:
Fixed